optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/bavet/bi/BavetGroupBiNode.java

```diff
@@ -40,6 +42,11 @@ public void addChildNode(BavetAbstractBiNode<GroupKey_, Result_> childNode) {
         childNodeList.add(childNode);
     }
 
+    @Override
+    public List<BavetAbstractBiNode<GroupKey_, Result_>> getChildNodes() {
```

Comment:
For consistency sake, please call this method `getChildNodeList()`.

The color of the curtains is pink, I know you hate it, but let's not mix colors of curtains - it's going to look ugly.